libbeat/template/processor.go

```diff
@@ -208,6 +209,9 @@ func (p *Processor) object(f *common.Field) common.MapStr {
 	}
 
 	switch f.ObjectType {
+	case "scaled_float":
+		dynProperties = p.scaledFloat(f)
+		addDynamicTemplate(f, dynProperties, matchType("*"))
```

Comment:
I use `*` by default because we want any type to match at least all numeric types.